### PR TITLE
Converting to AICS cookiecutter

### PR DESCRIPTION
**Pull request by gregjohnso**

### CODE REVIEW

File: .github/workflows/build-docs.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: Documentation
```
Comment (JacksonMaxfield) [inline .github/workflows/build-docs.yml]: You should try running `make docs` on this repo on your local machine. See if it actually works.

File: .github/workflows/build-master.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install .[test]
+    - name: Lint with flake8
+      run: |
+        flake8 fnet --count --verbose --max-line-length=127 --show-source --statistics
```
Comment (JacksonMaxfield) [inline .github/workflows/build-master.yml]: I think this may fail because Chek always had specific flake8 checks but we will see.

File: Makefile [hunk 3]
```diff
@@ -1,18 +1,59 @@
-# Ideas from "Automating Code Quality" talk by Kyle Knapp
+.PHONY: clean clean-test clean-pyc clean-build docs help
+.DEFAULT_GOAL := help
 
-check:
-	flake8 --extend-ignore=E501 --max-complexity 10 fnet tests examples
```
Comment (JacksonMaxfield) [inline Makefile]: Add these to our setup.cfg I believe

File: doc/source/conf.py [hunk 4]
```diff
@@ -1,174 +0,0 @@
-# -*- coding: utf-8 -*-
```
Comment (JacksonMaxfield) [inline doc/source/conf.py]: Again, try running `make docs` and see if it builds and launches the docs on your local machine but this file (or parts of it) may need to be copied to our conf.py

File: docs/index.rst [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+.. toctree::
+   :hidden:
+   :maxdepth: 1
+   :caption: Contents:
+
+   Overview <self>
+   installation
+   Package modules <modules>
+   contributing
+   math
```
Comment (JacksonMaxfield) [inline docs/index.rst]: Remove this line.

File: docs/installation.rst [hunk 6]
```diff
@@ -0,0 +1,51 @@
+.. highlight:: shell
```
Comment (JacksonMaxfield) [inline docs/installation.rst]: This is the standard installation doc but you may want to include something about CUDA and such.

File: docs/math.rst [hunk 7]
```diff
@@ -0,0 +1,9 @@
+=====================
```
Comment (JacksonMaxfield) [inline docs/math.rst]: Remove this file.

File: docs/installation.rst (hunk 6 above)
Comment (gregjohnso) [reply docs/installation.rst]: cuda gets automagically installed with Torch.

**Pull request merged**